packages/react-composites/src/composites/common/MeetingConferencePhoneInfoModal.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+import React from 'react';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { MeetingConferencePhoneInfoModal, ConferencePhoneInfo } from '@internal/react-components';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { TeamsMeetingAudioConferencingDetails } from '@azure/communication-calling';
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/** @private */
+export const MeetingConferencePhoneInfo = (props: {
```

Comment:
does this component serve a purpose or could we remove it and just directly use `MeetingConferencePhoneInfoModal`

Reply:
removed